src/main/java/com/renatusnetwork/momentum/gameplay/Scoreboard.java

```diff
@@ -178,7 +178,8 @@ else if (playerStats.inPracticeMode())
                 {
                     board.add(Utils.translate("&7"));
                     board.add(formatSpacing(Utils.translate("&6&lPracticing")));
-                    board.add(formatSpacing(Utils.translate("&e/prac &7to toggle")));
+                    board.add(formatSpacing(Utils.translate("&e/prac &7to set cp")));
```

Comment:
what about wording of: "&e/prac &7to set" and "&e/unprac &7to exit"?

Reply:
that makes more sense yeah

changed